Makefile

```diff
@@ -15,7 +15,6 @@ all: build
 # Clean everything up.
 clean:
 	rm -f core
-	rm -rf build
```

Comment:
The setup.py build creates this directory and grepping through the code recursively you'll get a bunch of hits from the copies of the source that are made there... it's a bit of a pain, that's why I delete it.

Now... can we delete it conditionally if some specific file in its contents is present? That would work

Reply:
I use `setup.py develop` so I think I don't run into this, or I missed it completely. Actually I think that spelling it `rm -rf build/` may be enough. Let me test and come back

Reply:
I was wrong, `setup.py build_ext` creates the `build` directory and I have a name clash if I don't remove the `BUILD` file before. This is unfortunate. I'll research a different solution.